### DISCUSSION TITLE
Signing Issue with google play console::: SDK version 33 requires a minimum of signature scheme v2

### DISCUSSION THREAD

**Discussion by idanasher** [airsdk/Adobe-Runtime-Support#661] [Q&A]
When I try to upload my new apk to the google play store, right after uploading the apk I get this error:

**You uploaded an APK with an invalid signature (learn more about signing). Error from apksigner: ERROR: MIN_SIG_SCHEME_FOR_TARGET_SDK_NOT_MET: Target SDK version 32 requires a minimum of signature scheme v2; the APK is not signed with this or a later signature scheme.**

I use air 33.1.1.345
publish with animate 2020 Version 2.0.1 (Build 19255)
and my certificate p12 file e was generated with animate long ago.
I have tried generating new certificates, and still get this error.

**Comment by marchbold** [comment]
Are you setting a target sdk of 32 in your manifest additions? You probably should only be using 29 at this point.

**Comment by idanasher** [comment]
Can i do that using all your latest ane's and latest air 33.1.1.345?

Thank you Michael
